src/styles/2-components/_card.scss

```diff
@@ -0,0 +1,53 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  background-color: $backgroundCardColor;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    margin-bottom: 4px;
+    max-width: 166px;
```

Comment:
You don't need max-width in every element as your "card" have a fixed width 200px and paddings: 32px 16px 16px;